website/docs/admin_colorspace.md

```diff
@@ -0,0 +1,193 @@
+
+
+#### Derived colorspace
+![distribution derived](assets/settings/admin_colorspace_distribution_3.png)
+
+When the host does not have any internal colorspace management, the colorspace distribution is derived from the file path. File rules are only used during transcoding or when creating reviewable files. When publishing, the resulting colorspace distribution is indicated by the [**colorspaceData**](dev_colorspace#data-model) key in the representation document.
+
+Since the host does not have any internal colorspace management, the loading process is ignoring [**colorspaceData**](dev_colorspace#data-model).
+
+
+## Configuration of global settings
+### Activating colorspace management on project
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_global_1.png)
+
+Colorspace management and distribution is disabled by default all project at the global level. Once it is enabled it is applied to all hosts. It is possible to override the global settings at the [host level](admin_colorspace#configuration-of-host-settings).
+
+:::warning Studio default
+
+Please be aware that activating the studio default project colorspace management will cause all projects to inherit the activation and this may potentially break some of the existing projects.
+
+:::
+
+### Config order
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_global_3.png)
+
+It's worth noting that the order in which the configuration paths are defined matters, with higher priority given to paths listed earlier in the configuration list. In the example image above, the production team decided to switch a project to use the `nuke-default` ([v1, linear, sRGB, rec709](https://opencolorio.readthedocs.io/en/latest/configurations/nuke_default.html)) OCIO config. This implies that all hosts will implement this config for publishing and loading within the [mentioned scope](admin_colorspace#how-does-colorspace-distribution-work-at-host-level).
+
+### Project level config
+![admin_colorspace_settings_global](assets/settings/admin_colorspace_settings_global_2.png)
+
+To avoid potential issues, ensure that the OCIO configuration path is not an absolute path and includes at least
+the root token (Anatomy). This helps ensure that the configuration path remains valid across different environments and
+avoids any hard-coding of paths that may be specific to one particular system.
+
+The example image above shows project level OCIO config file. This way the production is securing that any future changes on studio OCIO config file will not break any of old projects. It is recommended to use the project's OCIO config as starting point for all derivations of host level OCIO configs.
+
+### Activating file rules
+File rules are disabled by default. Once it is enabled it is applied to all hosts. It is possible to override them at the [host level](admin_colorspace#configuration-of-host-settings).
```

Comment:
After reading this I'm still in doubt whether a host specific setting can override the enabled state for the file rules. If I disable this checkbox in Global does Override file rules do anything inside the Host specific settings? I still get confused by this.

I feel like in settings this might also benefit a label that it's a global override to ALL file rules, both at the global level and the application settings.

It's funny because in **file rules override** below this IS explained. These two should **definitely** be merged into one. So that the statement:

> If global File rules are disabled, the host-level File rules are ignored.

Is here.